CMakeLists.txt

```diff
@@ -63,18 +68,18 @@ MACRO (ADD_PCT_EXECUTABLE EXE_NAME)
 ENDMACRO (ADD_PCT_EXECUTABLE)
 
 # Executables
-IF(NOT WIN32)
+IF(PCT_WITH_ROOT AND NOT WIN32)
   ADD_PCT_EXECUTABLE(pctpairprotons pctpairprotons.cxx)
   TARGET_LINK_LIBRARIES(pctpairprotons ${ROOT_LIBRARIES})
   ADD_PCT_EXECUTABLE(pctpairprotonsLomaLinda pctpairprotonsLomaLinda.cxx)
   TARGET_LINK_LIBRARIES(pctpairprotonsLomaLinda ${ROOT_LIBRARIES})
   ADD_PCT_EXECUTABLE(pctpairprotonsLMU_IMPCT pctpairprotonsLMU_IMPCT.cxx)
   TARGET_LINK_LIBRARIES(pctpairprotonsLMU_IMPCT ${ROOT_LIBRARIES})
-ENDIF(NOT WIN32)
-ADD_PCT_EXECUTABLE(pctpaircuts pctpaircuts.cxx)
-TARGET_LINK_LIBRARIES(pctpaircuts ${ROOT_LIBRARIES} RooFit RooFitCore Minuit)
-ADD_PCT_EXECUTABLE(pctlluconverter pctlluconverter.cxx)
-TARGET_LINK_LIBRARIES(pctlluconverter ${ROOT_LIBRARIES})
+  ADD_PCT_EXECUTABLE(pctpaircuts pctpaircuts.cxx)
```

Comment:
It's good to remove the root dependency but we can't entirely disable pctpaircuts. Would be nice to have a compilation define (e.g., `PCT_USE_ROOT`) to disable certain features in pctpaircuts.

Reply:
I have implemented such idea atop `master` here: https://github.com/acoussat/PCT/commit/5ae18f51d8830f36a8b98f1abe4a4c77030d380e. Should I open a new PR with this code or update this PR?

Reply:
No, I think it's part of the same PR.
Nothing wrong with your code but I'd consider creating a `pctConfiguration.h.in` similar to [RTK's `rtkConfiguration.h.in`](https://github.com/RTKConsortium/RTK/blob/master/rtkConfiguration.h.in). What do you think?